docs/sources/reference/components/prometheus/prometheus.exporter.ssh.md

```diff
@@ -0,0 +1,205 @@
+---
+canonical: https://grafana.com/docs/alloy/latest/reference/components/prometheus/prometheus.exporter.ssh/
+aliases:
+  - ../prometheus.exporter.ssh/ # /docs/alloy/latest/reference/components/prometheus.exporter.ssh/
+description: Learn about prometheus.exporter.ssh
+title: prometheus.exporter.ssh
+---
+
+# prometheus.exporter.ssh
+
+The `prometheus.exporter.ssh` component embeds an SSH exporter for collecting metrics from remote servers over SSH and exporting them as Prometheus metrics.
+
```

Comment:
Please add the experimental snippet to document the fact that the component is experimental